task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -492,9 +492,13 @@ export default {
             type: 'success',
             message: this.$t('alert.success.task.startDate')
           });
+          this.$root.$emit('task-start-date-updated', this.task);
         }).catch(e => {
```

Comment:
```suggestion
        }).catch(() => {
```
`e` variable is no more used.